governance/handler.go

```diff
@@ -253,6 +306,22 @@ func checkBigInt(k string, v interface{}) bool {
 }
 
 func checkAddress(k string, v interface{}) bool {
```

Comment:
Function name is very ambiguous.
something like checkAddressOrListOfUniqueAddresses ?

Reply:
Reflected at https://github.com/klaytn/klaytn/pull/1110/commits/25b86fa38cffb37df13e62505554b020e7daf6a6.